src/components/nav_item.tsx

```diff
@@ -10,11 +10,13 @@ interface NavLinkProps extends React.HTMLProps<HTMLAnchorElement> {
 const StyledLink = styled.a`
   align-self: center;
   font-family: ${ fonts.primary.fontFamily };
-  font-size: 15px;
+  font-size: 14px;
   line-height: 12px;
   text-decoration: none;
   color: black;
   text-transform: uppercase;
+  -webkit-font-smoothing: antialiased;
```

Comment:
`font-smoothing` is missing from the list of properties to that need to be prefixed in `styled-components`. I didn't get around to submitting a PR to styled-components to fix it yet but I will for sure